backend/coins/eth/account.go

```diff
@@ -438,6 +440,7 @@ func (account *Account) FatalError() bool {
 // Close implements accounts.Interface.
 func (account *Account) Close() {
 	defer account.initializedLock.Lock()()
+	account.quitChan <- struct{}{}
```

Comment:
this seems unnecessary because we close the channel later anyway, but the problem is the `account.quitChan = nil` as the channel might be read at the same time. I guess moving the `close(account.quitChan)` up here and leaving the `..= nil` where it is, makes more sense

Reply:
actually just removing the `account.quitChan = nil` would be the best solution imo. I don't think it affects how the struct is being garbage collected anyway

Reply:
as you state, this is the right solution to the problem. this is blocking operation until some goroutine reads it. close(...) broadcasts to all listening goroutines instead.

use a RWMutex on quitChan instead to resolve the race condition.

Reply:
I used the `locker.Lock` for the convenient `defer` locking/closing.